data/product.d/scientific-linux.conf

```diff
@@ -8,6 +8,7 @@ product_name = Red Hat Enterprise Linux
 
 [Payload]
 enable_updates = True
+enable_addon_repos = False
```

Comment:
Please, change the name of the option to `enable_repositories_from_treeinfo`.

Reply:
I'll make this change shortly.